pixi.toml

```diff
@@ -95,6 +95,7 @@ botocore = "*"
 s3fs = ">=2022.1"
 adlfs = ">=2022.1"
 gcsfs = ">=2022.1"
+huggingface_hub = "*"
```

Comment:
Should we add this to the pyproject.toml as an optional, so `pip install bodo[huggingface]` works? Or something like that

Reply:
Done, thanks for the suggestion.